src/stratis_cli/_error_reporting.py

```diff
@@ -125,6 +126,10 @@ def _interpret_errors(errors):
                 "the D-Bus: %s."
             )
             return fmt_str % error
+
+        if isinstance(error, StratisCliUserError):
+            fmt_str = "Invalid user input: %s"
```

Comment:
I'm not sure I'ld go with this, because the input _is_ valid, it's the context that makes us decide that it is wrong. I'ld go with something gentler and more explanatory, "It looks like you issued a command that you did not intend", maybe.